server/database/dbOperations.ts

```diff
@@ -248,3 +251,397 @@ export const removeTableFromConfig = async (
     });
   });
 };
+
+// Annotated Collections Operations
```

Comment:
As this file is getting quite large I wonder if we should move these to an `annotatedCollections.ts` file, and keep this file focused on general database utilities.